frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -151,16 +135,34 @@ function ReaderPaging:setupTapTouchZones()
     })
 end
 
--- This method will be called in onSetDimensions handler
 function ReaderPaging:setupTouchZones()
-    -- deligate gesture listener to readerui
     self.ges_events = {}
     self.onGesture = nil
-
     if not Device:isTouchDevice() then return end
 
-    self:setupTapTouchZones()
+    local forward_zone, backward_zone = self.view:getTapZones()
+
     self.ui:registerTouchZones({
+        {
+            id = "tap_forward",
+            ges = "tap",
+            screen_zone = forward_zone,
+            handler = function()
```

Comment:
You're duplicating this in ReaderPaging (it's already there in setupTapTouchZones() ).
But you haven't duplicated it in ReaderRolling.
Is this willingly ?
Any reason for this difference ? Can't it be deduplicated too?

Reply:
You mean in a commit you yet have to push?
(Because I see it twice in current https://github.com/koreader/koreader/blob/b8fdd5ebf8adbfbfe107b505f84b3f05697b2171/frontend/apps/reader/modules/readerpaging.lua#L110-L167)